src/database/migrations/20191001134213-create-user.js

```diff
@@ -8,8 +8,18 @@ module.exports = {
         primaryKey: true,
         type: Sequelize.INTEGER
       },
-      example: {
-        type: Sequelize.STRING
+      username: {
+        type: Sequelize.STRING,
+        allowNull: false,
+      },
+      password: {
+        type: Sequelize.STRING,
+        allowNull: false,
+      },
+      email: {
+        type: Sequelize.STRING,
+        allowNull: false,
+        unique: true,
       },
       createdAt: {
```

Comment:
you can set the default value of these fields(`createdAt`, `updatedAt`) like `defaultValue: Sequelize.NOW` just make it easy